src/chains/filecoin/filecoin/src/blockchain.ts

```diff
@@ -238,6 +235,34 @@ export default class Blockchain extends Emittery.Typed<
     return this.ipfsServer.node;
   }
 
+  enableMiner() {
+    this.#minerEnabled = true;
+    this.emit("minerEnabled", true);
+
+    if (this.options.miner.blockTime > 0) {
+      const intervalMine = () => {
+        this.mineTipset();
+      };
+
+      this.miningTimeout = setInterval(
```

Comment:
We should use a recursive `setTimeout` implementation; see https://github.com/trufflesuite/ganache-core/pull/836#discussion_r595207331